index.html

```diff
@@ -0,0 +1,131 @@
+                 </div>
+              </div>
+    </header>
+
+<section class="introduction">
+    <h2>"Hello! Sharing World"</h2>
+    <h1>WOMEN IN TECH GLOBAL CONFERENCE 2023</h1>
+    <p class="info">  ​​​​​​The hybrid conference will bring women in tech, minorities and allies from all over the world together 
+        through an interactive platform featuring live educational & training content, keynotes, engaging panels, breakout rooms, 
+        technical workshops, and networking with virtual and in-person sessions.
+    </p>
+    <p class="date">2023.05.9(TUE) ~ 11(THU)</p>
+    <p class="location">The three main summits are virtual. In person locations will be require a separate invite</p>
+</section>
+
+   <section class="programs">
+   <h3>Main Program</h3>
+   <hr>
+   <div class="program-list">
+    <div class="lecture">
+        <i class='fa-solid fa-person-dress'></i>
+        <h5>Lecture</h5>
+        <p>Listen to speakers form various countries about the messages of sharing and opening</p>
+    </div>
+    <div class="discussion">
+        <i class='fa fa-users'></i>
+        <h5>Discussion</h5>
+        <p>Discussions with feature leading technology experts and professionals from all over the world.</p>
+    </div>
+    <div class="workshop">
+        <i class='fa-solid fa-pen-to-square'></i>
+        <h5>Workshop</h5>
+        <p>Talk to women's, minorities and allies working in tech or considering tech career.</p>
+    </div>
+    <div class="sharing">
+        <i class='fa-solid fa-chalkboard-user'></i>
+        <h5>Forum</h5>
+        <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+    </div>
+    <div class="network">
+        <i class='fas fa-globe-americas'></i>
+        <h5>Community</h5>
+        <p>Join the communities from across the globe through smaller concurrent virtual events.</p>
+    </div>
+ <button type="submit" class="button-1">Join Women In Tech 2023</button>
+   </div>
+   <a class="link-1" href="#">SEE THE WHOLE PROGRAM</a>
+   </section> 
+
+   <section class="speakers">
+     <h2 class="sp-lists">Featured Speakers</h2>
+     <hr>
+     <div class="gridContainer"></div>
+   </section>
+
+   <section class="partners">
+    <h2>Partner</h2>
+    <hr>
+    <div class="partners-list">
+    <a rel="noopener noreferrer" target="_blank" href="https://www.mozilla.org/en-US/"><img src="img/mozilla.svg" alt="Mozilla"></a>
+    <a rel="noopener noreferrer" target="_blank" href="https://www.google.com/en-US/"><img src="img/google.svg" alt="Google"></a>
+    <a rel="noopener noreferrer" target="_blank" href="https://www.naver.com/en-US/"><img src="img/naver.svg" alt="Naver"></a>
+    <a rel="noopener noreferrer" target="_blank" href="https://www.daumkakao.com/en-US/"><img src="img/daumkakao.svg" alt="Daumkakao"></a>
+    <a rel="noopener noreferrer" target="_blank" href="https://www.airbnb.com/en-US/"><img src="img/airbnb.svg" alt="Airbnb"></a>    
+</div>
+</section>
+

```

Comment:
This section should be visible on the mobile screen also. Currently, it is only visible in desktop mode only. The alignment of individual elements should be vertical.

Reply:
Okay, thank you for reviewing my project. I'll correct it.